modules/custom/d_p/d_p.module

```diff
@@ -74,7 +70,7 @@ function d_p_preprocess_paragraph(&$variables) {
         continue;
       }
 
-      $settings = json_decode($field->first()->get('value')->getValue());
+      $settings = $field->first()->getValue();
 
       if (!isset($settings->custom_class) || !isset($settings->custom_theme_colors)
         || strpos($settings->custom_class, 'theme-custom') === FALSE
```

Comment:
Maybe we could handle this in a new way? To avoid this strpos

Reply:
Yes, this should be doable, I can rework it to use newly introduced methods.

Reply:
I've reworked this part a little bit, you can review it again